docs/learn/learn-async-backing.md

```diff
@@ -7,103 +7,253 @@ keywords: [parachains, backing, parablock, perspective parachains, unincluded se
-parachain collators to fit more transactions and prepare block candidates for backing and inclusion.
-**Contextual execution** refers to how a parablock can be built earlier using the context provided
-by an _unincluded segment_ of recent block ancestors.
+:::info Every collator also runs an attached relay chain full node
 
-**Unincluded segments** are chains of candidate blocks that are not yet included in the relay chain.
-Parablocks can be added to this unincluded segment without waiting for the latest included parent
-block of the relay chain. The core functionality that asynchronous backing brings is the ability to
-build on these unincluded segments of block ancestors rather than ancestors included in the relay
-chain state.
+The attached relay node receives relay blocks via gossip. Then the relay node talks to the parachain
+node through the `CollationGeneration` subsystem. R2 is gossiped to the relay full node attached to
+the collator producing P2. Then CollationGeneration passes information about R2 to the collator
+node. Finally, relay parent information from R2 informs the generation of candidate P2.
 
-Currently, parablocks rely on the most recent relay chain block (often referred to as the **parent**
-block, as the parablock anchors itself to it). Each parablock must be generated and go through the
-entire backing process in a single relay block. It then proceeds to the availability + inclusion
-process during the next block.
+:::
+
+Because P2 is rushing to be backed in 6 seconds into R3, collators have only 0.5 seconds to generate
+it and present it to backing groups on the relay chain that will take approximately 2 seconds to
+back it and some extra time for gossiping it (the whole process from collation to backing lasts 6
+seconds). P2 is included in R4, which could be used as a relay parent for P3 (not shown). After 24
+seconds P1 and P2 are included into the relay chain. Note how collators can start new parablocks
+every 12 seconds, but only have 0.5 seconds to generate them.
 
-:::info
+## Asynchronous Backing
 
-It is important to note the following clarifications:
+:::warning Disclaimer: Performance Measurements
 
-1. The parablock is not placed on the relay chain in its entirety, but rather the **paraheader**
-2. The relay chain does not access the entire state of a parachain but only the values that changed
-   during that block and the merkelized hashes of the unchanged values.
+Due to asynchronous backing not being fully implemented in a running production network, each
+performance metric is not thoroughly tested nor guaranteed until proper benchmarking has occurred.
 
 :::
 
-For more information on the validity and availability process, be sure to visit the
-[parachain protocol](../learn/learn-parachains-protocol.md) page.
+### Synchronous vs. Asynchronous Backing
+
+Below, a table showing the main differences between synchronous and asynchronous backing.
```

Comment:
i might softly suggest you start with the new graphic rather than this table to be the first thing users see to explain the difference between async and sync backing.

perhaps exactly the image i linked to from Sophia.

That at least anchors them to the existing reading, and then a table showing the benefits can naturally follow.